src/main/java/edu/harvard/iq/dataverse/ConfigureFragmentBean.java

```diff
@@ -35,15 +41,21 @@
     private ExternalTool tool = null;
     private Long fileId = null;
     private ExternalToolHandler toolHandler = null;
+    private String messageApi = "";
     
     @EJB
     DataFileServiceBean datafileService;
     @Inject
     DataverseSession session;
     @EJB
     AuthenticationServiceBean authService;
+    @EJB
+    UserNotificationServiceBean userNotificationService;
     
-    public String configureExternalAlert() { 
+    public String configureExternalAlert() {
+        generateApiToken();
+        String httpString = "window.open('" + toolHandler.getToolUrlWithQueryParams()+  "','_blank'" +")";
+        RequestContext.getCurrentInstance().execute(httpString);
```

Comment:
I think this is deprecated. 
PrimeFaces.current().executeScript(httpString);
should do the same thing...

Reply:
You are right. I replaced RequestContext with PrimeFaces as you recomended.